client/src/inside/components/EhrContextBanner.vue

```diff
@@ -10,19 +10,21 @@
               fas-icon(icon="plus", v-show="!show") 
               fas-icon(icon="minus", v-show="show")
         div(v-show="show")
-          ehr-context-instructor(v-if="showInstructor")
-          ehr-context-developer(v-if="showSeeding")
+          ehr-context-read-only-instructor(v-if="isReadOnlyInstructor")
+          ehr-context-instructor(v-else-if="showInstructor")
```

Comment:
Merge the read only into the instructor context and use a "read only" to create the readonly behaviour.